data/operators/amenity/post_office.json

```diff
@@ -25,6 +25,37 @@
     }
   },
   "items": [
+    {
+      "displayName": "DTDC",
+      "locationSet": {"include": ["in"]},
+      "matchNames": ["Desk to Desk Courier & Cargo"],
+      "tags": {
+        "amenity": "post_office",
+        "brand": "Desk to Desk Courier & Cargo",
+        "brand:short": "DTDC",
+        "brand:wikidata": "Q55608802",
+      }
+    },
+    {
+      "displayName": "Blue Dart",
+      "locationSet": {"include": ["in"]},
+      "matchNames": ["Desk to Desk Courier & Cargo"],
```

Comment:
It looks like DTDC and Blue Dart have the same matchName, which might cause conflicts